sdk/core/azure-core/test/e2e/azure_core_storage_list_containers_sample.cpp

```diff
@@ -1,9 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
-#include "azure/core/http/curl/curl.hpp"
-#include "azure/core/http/http.hpp"
-#include "azure/core/http/pipeline.hpp"
+#include <azure/core/http/curl/curl.hpp>
+#include <azure/core/http/http.hpp>
+#include <azure/core/http/pipeline.hpp>
```

Comment:
?  isn't it the other way around?  or why this case is different?

Reply:
Presumably because it is a sample/test.

Reply:
Yes, that mirrors the way our customers are most likely to include our headers

> Updated tests to mirror how a customer will include the header
> 
> #include <>

Reply:
It makes sense for samples to have `<>`. 

However, I am wondering, why did you decide to do the same for tests too?